spec/rspec/core/resources/inconsistently_ordered_specs.rb

```diff
@@ -0,0 +1,12 @@
+# Deliberately named _specs.rb to avoid being loaded except when specified
```

Comment:
I did `_specs.rb` here to follow `spec/rspec/core/resources/formatter_specs.rb` which was already in place.

Later I added the other file, and initially did the `_specs.rb` thing there, too, but that caused a failure in our "load all files with warnings enabled to see if any warnings are emitted" spec.  That other file has a load-time error so being loaded by the warnings check caused a failure.  Using `.rb_` prevented it from being included in that check.

Any alternate suggestions?  I could use an environment variable to make that file call `contex` only when the caller tells it to via the env var.